clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1660,23 +1658,26 @@ class SyclKernelDeclCreator : public SyclKernelFieldHandler {
 class SyclKernelBodyCreator : public SyclKernelFieldHandler {
   SyclKernelDeclCreator &DeclCreator;
   llvm::SmallVector<Stmt *, 16> BodyStmts;
+  llvm::SmallVector<InitListExpr *, 16> CollectionInitExprs;
   llvm::SmallVector<Stmt *, 16> FinalizeStmts;
-  llvm::SmallVector<Expr *, 16> InitExprs;
+  llvm::SmallVector<std::pair<InitializedEntity, uint64_t>, 8> ArrayInfos;
```

Comment:
It is not obvious what uint64_t part of pair holds, could you please add a comment?